packages/fuzzer/utils.cpp

```diff
@@ -18,18 +18,19 @@
 #include <csignal>
 #include <iostream>
 
-void StartLibFuzzer(const std::vector<std::string> &args,
-                    fuzzer::UserCallback fuzzCallback) {
+void StartLibFuzzer(const std::vector<std::string> &args, fuzzer::UserCallback fuzzCallback) {
   std::vector<char *> fuzzer_arg_pointers;
-  for (auto &arg : args)
-    fuzzer_arg_pointers.push_back((char *)arg.data());
+  for (const std::string &arg : args) {
+    fuzzer_arg_pointers.push_back(const_cast<char *>(arg.c_str()));
```

Comment:
Why make the args const and then cast the const away in the next line?
.c_str() and .data() are equivalent since c++11.
Also, the fuzzer is only started once, and there are usually not that many parameters.